index.html

```diff
@@ -14,32 +14,60 @@
   <body>
     <nav class="top-nav">
       <a href="#" class="logo-txt"> Makoji David </a>
-      <div class="menu">
+      <div class="menu" id="hamburger">
         <img
-          src="./img/icons/hamburger.png"
+          src="./img/icons/hamburger.svg"
           alt="logo-icon"
           class="logo-icon"
         />
       </div>
       <ul class="top-menu">
         <li class="top-menu-item motion">
-          <a href="#" class="links">portfolio</a>
+          <a href="#portfolio" class="links">portfolio</a>
         </li>
         <li class="top-menu-item motion">
-          <a href="#" class="links">about</a>
+          <a href="#about" class="links">about</a>
         </li>
         <li class="top-menu-item motion">
-          <a href="#" class="links">contact</a>
+          <a href="#contact" class="links">contact</a>
         </li>
         <li class="top-menu-item motion">
           <a href="#" class="links"
             ><img src="./img/icons/email.png" alt="mail-icon"
           /></a>
         </li>
       </ul>
+      <div class="menu-container" id="topMenu">
+        <div class="close" id="close">
+          <img src="./img/icons/close_white_48dp.svg" alt="close-icon" />
+        </div>
```

Comment:
- [ ] please make sure that the close button has the same size as the one in the Figma design and also try to align it in the right place as in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/179769767-4486bd0c-406d-4801-ae67-bc9e528a3928.png)